spec/rails_helper.rb

```diff
@@ -119,108 +97,131 @@ def stub_addresses
   # Make FactoryBot easier.
   config.include FactoryBot::Syntax::Methods
 
-  # Stub out Geocoder or else...
-  config.before(:all) do
-    stub_addresses
+  #
+  # --------------------
+  # START - Seeding helpers for tests setup
+  # --------------------
+  #
+
+  def default_records_data
+    {
+      organizations: {
+        default: {
+          name: "DEFAULT"
+        }
+      },
+      users: {
+        organization_admin: {
+          name: "ORG ADMIN"
+        },
+        user: {
+          name: "USER"
+        },
+        super_admin: {
+          name: "SUPER ADMIN"
+        },
+        super_admin_no_org_name: {
+          name: "SUPER ADMIN NO ORG NAME"
+        }
+      },
+      partners: {
+        default: {
+          name: "DEFAULT PARTNER"
+        }
+      }
+    }
   end
 
-  # set driver for system tests
-  config.before(:each, type: :system) do
-    clear_downloads
-    driven_by :chrome
-    Capybara.server = :puma, { Silent: true }
-  end
+  def define_global_variables
+    @organization = Organization.find_by!(name: "DEFAULT")
 
-  config.after(:each, type: :system, js: true) do
-    clear_downloads
+    user_names = default_records_data[:users].map { |k, v| v[:name] }.compact
+    users = User.where(name: user_names)
+    @organization_admin = users.find { |u| u.name == default_records_data[:users][:organization_admin][:name] }
+    @user = users.find { |u| u.name == default_records_data[:users][:user][:name] }
+    @super_admin = users.find { |u| u.name == default_records_data[:users][:super_admin][:name] }
+    @super_admin_no_org = users.find { |u| u.name == default_records_data[:users][:super_admin_no_org_name][:name] }
+
+    @partner = Partner.find_by!(name: default_records_data[:partners][:default][:name])
```

Comment:
I'm not sure we're gaining a lot with the `default_records_data`. There are only 6 values we're using - storing them in a deeply nested hash is kind of confusing. I'd rather just have those 6 values as constants.

Reply:
Sounds good, I was kind of wrestling with it. But yeh now that you say that I think using a constant makes sense.